icaruscode/CRT/CRTDataAnalysis_module.cc

```diff
@@ -123,28 +123,28 @@ namespace crt {
         };
    
       fhicl::Atom<art::InputTag> TriggerLabel {
-        Name("TriggerLabel"),
-	  Comment("Label for the Trigger fragment label")
-	  };
+       Name("TriggerLabel"),
+	      Comment("Label for the Trigger fragment label")
+	      };
 
       fhicl::Atom<double> QPed {
-	Name("QPed"),
-	  Comment("Pedestal offset [ADC]")
-	  };
+	     Name("QPed"),
+	      Comment("Pedestal offset [ADC]")
+	      };
       fhicl::Atom<double> QSlope {
-	Name("QSlope"),
-	  Comment("Pedestal slope [ADC/photon]")
-	  };
+	     Name("QSlope"),
+	      Comment("Pedestal slope [ADC/photon]")
+	      };
 
       fhicl::Atom<double> PEThresh {
-	Name("PEThresh"),
-	  Comment("threshold in photoelectrons above which charge amplitudes used in hit reco")
-	  };
-
-      fhicl::Atom<uint64_t> CrtWindow {
-	Name("CrtWindow"),
-	  Comment("window for looking data [ns]")
-	  };
+	     Name("PEThresh"),
+	      Comment("threshold in photoelectrons above which charge amplitudes used in hit reco")
+	      };
+
+     fhicl::Atom<uint64_t> CrtWindow {
+	     Name("CrtWindow"),
+	      Comment("window for looking data [ns]")
+	      };
```

Comment:
Code indentation looks strange.
Please reindent consistently with the rest (it should be space only, 2 spaces per indentation level).